src/components/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { Input } from '@material-ui/core'
+
+const inputStyle = {
```

Comment:
If it is not overriding of common input styles it's better to use material theme values or just configure input on theme creation

Reply:
I can remove it, but now default inputs don't have borders & have white color.

Reply:
I mean - if any input is assumed to have these styles, I'd rather apply them in global config like
https://material-ui.com/customization/globals/
If this input is unique and exists only inside Search, then it's ok.